MEP_Engine/Modify/SetGeometry.cs

```diff
@@ -54,6 +56,22 @@ public static IFlow SetGeometry(this IFlow flowObj, ICurve curve)
             return clone;
         }        
 
+        
+        /***************************************************/
+        
+        public static Fitting SetGeometry(this Fitting fittingObj, Point location, List<Point> connectionLocations)
+        {
+            if (connectionLocations.Count < 2)
+            {
+                Engine.Reflection.Compute.RecordError("A fitting requires at least 2 connections, e.g. an elbow fitting has 2.");
```

Comment:
Simply saying "e.g. an elbow fitting has 2" does not tell the user `why` an elbow has 2 connections. Expand this error message to help the user work out what they need to be doing and why.